src/commands/NewCamelRouteCommand.ts

```diff
@@ -45,4 +53,15 @@ export class NewCamelRouteCommand extends AbstractNewCamelRouteCommand {
 			await this.showNoWorkspaceNotification();
 		}
 	}
+
+	private async showQuickPickForCamelRouteDSL(): Promise<QuickPickItem | undefined> {
+		const items: QuickPickItem[] = [
+			{ label: 'YAML', description: 'YAML DSL' },
+			{ label: 'XML', description: 'XML DSL' },
```

Comment:
The description doesn't provide more information. and in the UI, it appears very next to the item, so it reada bit like "YAML YAML DSL".  I think we can remove it or provide a more precise description.

For instance in VS COde Langaueg Support, what is used is: `YAML DSL` and description : `Camel Route using YAML DSL`